full-moon/src/tokenizer.rs

```diff
@@ -15,6 +15,8 @@ use serde::{Deserialize, Serialize};
 use std::{borrow::Cow, cmp::Ordering, fmt, str::FromStr};
 
 symbols!(
+    // TODO: This only is valid in Roblox
+    Ampersand => "&",
```

Comment:
By alphabetical order with the rest of the group, I mean with the rest of the single character symbols.